content/en/docs/concepts/what-is-istio/index.md

```diff
@@ -45,7 +45,13 @@ between microservices, then configure and manage Istio using its control plane f
 
 * Secure service-to-service communication in a cluster with strong identity-based authentication and authorization.
 
-Istio is designed for extensibility and meets diverse deployment needs.
+Istio is designed for extensibility and meets diverse deployment needs. The following [architecture](/docs/ops/deployment/architecture/) diagram shows the various components:
+
+{{< image width="80%"
```

Comment:
It seems strange to show the exact same picture as what you see if you click on the link just above it.

Reply:
Well, clicking on the link shows the diagram plus description of the components :)  Ideally, we want the diagram in /docs/ops/deployment/architecture/ to show more details, like gateways, certs, wasm, etc

I think the duplication is better than not having a diagram on this page..wdyt?

Reply:
@frankbu Personally, when I was looking at the docs on community testing day, I though it very odd that we didn't have a picture on the `What is Istio?` page and wondered if we lost it somewhere.

I'm not against using the same picture.